src/main/java/edu/tamu/scholars/middleware/messaging/handler/CustomStompSubProtocolErrorHandler.java

```diff
@@ -12,9 +12,9 @@ public class CustomStompSubProtocolErrorHandler extends StompSubProtocolErrorHan
 
     @Override
     protected Message<byte[]> handleInternal(StompHeaderAccessor errorHeaderAccessor, byte[] errorPayload, @Nullable Throwable cause, @Nullable StompHeaderAccessor clientHeaderAccessor) {
-    	if (Objects.nonNull(clientHeaderAccessor)) {
-    		errorHeaderAccessor.setDestination(clientHeaderAccessor.getDestination());
-    	}
+        if (Objects.nonNull(clientHeaderAccessor)) {
+            errorHeaderAccessor.setDestination(clientHeaderAccessor.getDestination());
+        }
```

Comment:
This is unusual request - the indentation  on Lines 15 and 17 are highlighted - although only Line 16 has the intended change.

Reply:
<img width="410" alt="image" src="https://github.com/TAMULib/scholars-discovery/assets/8352733/8a4dded0-c402-450e-93ea-4138a004427a">

LGTM